server/mergin/commands.py

```diff
@@ -117,6 +127,12 @@ def _check_server():  # pylint: disable=W0612
         else:
             click.secho("Database initialized properly", fg="green")
 
+        if edition == "Community Edition":
```

Comment:
we usually do not do this kind of if-else branches as it is difficult to maintain
usually in other versions we have overloaded/extended functions which do the right job

we can only check data dir for sync as it is the most critical and common here